spec/classes/aerospike_spec.rb

```diff
@@ -437,7 +453,10 @@
 				# Tests related to the aerospike::config class
 
 				# Tests related to the aerospike::service class
-				it { is_expected.to_not contain_service('amc') }
+				it { is_expected.to contain_service('aerospike').with_hasrestart(true)\
```

Comment:
Why dropping `it { is_expected.to_not contain_service('amc') }` ? It is important to keep tests that ensure that some services are not present in addition to test that some are present.